app/src/lib/get-os.ts

```diff
@@ -16,3 +16,24 @@ export function getOS() {
     return `${OS.type()} ${OS.release()}`
   }
 }
+
+/** See the OS we're currently running on is at least Mojave. */
+export function isMojaveOrLater() {
+  if (__DARWIN__) {
+    const parser = new UAParser()
+    const os = parser.getOS()
+
+    if (os.version == null) {
```

Comment:
`os.version == null` should be `os.version === undefined` since
`os.version` is typed as `string | undefined` and we tend to favor explicit checks.